Data_Engine/Query/Count.cs

```diff
@@ -20,32 +20,57 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
+using BH.oM.Base.Attributes;
 using BH.oM.Data.Collections;
 using System;
+using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
 
 namespace BH.Engine.Data
 {
     public static partial class Query
     {
+        /***************************************************/
+        /**** Interface Methods                         ****/
+        /***************************************************/
+
+        [PreviousVersion("6.0", "BH.Engine.Reflection.Query.ICount(System.Collections.Generic.List<System.Object>)")]
+        [Description("Get the count of the number of items in a list of objects.")]
+        [Input("list", "The list of items to obtain a count of.")]
+        [Output("count", "The number of items in the list.")]
+        public static int ICount<T>(this List<T> list)
```

Comment:
Can't we just remove this method, and version it to the other one added on line 98.

Appreciate you are just moving things across, and already done some sensible clean-up, but think this additional step also could be done?

This is unless I am missing some obvious reason as to why we need to interface method, such as some difference in behaviour in something like Excel or similar, but have a hard time seeing that?